### ISSUE TITLE
[repeat] [input box] Vimium-C command isn't usable in WhatsApp Web chatbox

### ISSUE THREAD

**Issue by gdh1995** [gdh1995/vimium-c#405]
> Refer: #400

**Describe the bug**
In [WhatsApp Web](https://web.whatsapp.com/), if the chatbox is focused, vimium-c isn't able to be triggered again unless using the mouse to click on any place to allow vimium-c gaining focus again. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to URL [WhatsApp Web](https://web.whatsapp.com/)
2. Click on any chat
3. `gi` to the chatbox
4. try to `esc`
5. nothing will happen like how other input boxes will be escaped
6. click on any blank place in the page
7. try `esc` again
8. the vimium c command is usable again

**Browser, OS and Vimium C version**

Browser: Edge 92.0.902.67 
OS: windows 10 21h1
Vimium C: 1.90.2

**Comment by zim0369** [comment]
Yes, I face this issue as well, so I avoid using whatsapp on desktop. But, if this gets fixed then life would be much better.

**Comment by gdh1995** [comment]
Sad. I can not login WhatApp - my account was deleted by the system for unknown reasons.

**Comment by Vaisakhkm2625** [comment]
I got a solution, just press <kbd>Tab</kbd>  key while it's focused on chat box... it will get out of there and we can use vimium normally

**Comment by QiBaobin** [comment]
Same behavior while I uses jira here

**Comment by patrykk21** [comment]
For similar websites we could implement a command with a key combination based on modifier keys to force the unfocus.
<ctrl-U> - force unfocus so vimium keybinds are working normally

**Comment by patrykk21** [comment]
I'd like to add that even when you manage to use vim commands on the browser then you still can't focus a specific chat :D

**Comment by gdh1995** [comment]
You may try holding on the `Escape` key to tell Vimium C to "try to remove a current focus".

As for how to select a chat item, you may need https://github.com/gdh1995/vimium-c/wiki/Add-or-remove-hints

**Comment by gusallc** [comment]
any news for this bug ?

Thanks.

**Comment by Vaisakhkm2625** [comment]
> any news for this bug ?
> 
> Thanks.

I haven't fount any solution other than pressing tab.... so now i am using surfingkeys with for me felt infinity better than vimium...

**Comment by Redoxahmii** [comment]
I can't even select a conversation even on first load
